packets.go

```diff
@@ -587,14 +597,6 @@ func (mc *mysqlConn) handleErrorPacket(data []byte) error {
 		return driver.ErrBadConn
 	}
 
-	pos := 3
-
-	// SQL State [optional: # + 5bytes string]
-	if data[3] == 0x23 {
-		//sqlstate := string(data[4 : 4+5])
-		pos = 9
-	}
-
 	// Error Message [string]
 	return &MySQLError{
 		Number:  errno,
```

Comment:
We need to add `sqlstate` here, shouldn't we?

Reply:
Got left out while preparing the PR. Thank you!